tslint.json

```diff
@@ -1,6 +1,6 @@
 {
     "rules": {
-        "tslint-no-unused-expression-chai": true,

```

Comment:
We edited this in a recent PR(https://github.com/OmniSharp/omnisharp-vscode/pull/2092). Why are we changing it here ?

Reply:
the syntax here is peculiar and the previous PR did not fully enable this linting rule override.